cpp/src/arrow/c/bridge_test.cc

```diff
@@ -740,6 +741,7 @@ TEST_F(TestArrayExport, Primitive) {
   TestPrimitive(large_utf8(), R"(["foo", "bar", null])");
 
   TestPrimitive(decimal(16, 4), R"(["1234.5670", null])");
+  TestPrimitive(decimal256(16, 4), R"(["1234.5670", null])");
```

Comment:
Can you also add import tests?

Reply:
done, added additional schema tests and a round trip test.